src/commands/create.js

```diff
@@ -164,8 +164,8 @@ export default asyncCommand({
 			//globally replace instances of {{name}} with value of name arg
 			['name'].forEach(str => {
```

Comment:
As this is executed only once, should we remove the loop itself?

Reply:
there is actually comment about this
`// TODO: concat author-driven patterns`
maybe plan is to add more placeholders